samples/js/dropbox/find-photos/src/main.js

```diff
@@ -126,7 +128,15 @@ async function main(): Promise<void> {
   });
 }
 
-function getSizes(input: Object): Map<Struct, string> {
+async function getSizes(resources: Map<Struct, Struct>): Promise<Map<Struct, string>> {
+  const tuples = [];
+  await resources.forEach((v: any, k: Struct) => {
+    tuples.push([k, v.url]);
+  });
+  return Promise.resolve(new Map(tuples));
```

Comment:
You don't have to manually wrap `new Map(tuples)` here. It is implied in an `async` function.